modules/baremetal-hardware-event-proxy-operator.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * operators/operator-reference.adoc
+[id="baremetal-hardware-event-proxy_{context}"]
+= Baremetal Hardware Event Proxy Operator
+
+[discrete]
+== Purpose
+The OpenShift Baremetal Hardware Event Proxy Operator manages the life-cycle of the Baremetal Hardware Event proxy. The Baremetal Hardware Event Proxy enables the user to configure the types of event that are monitored using Redfish Hardware Events on the cluster.
+
+== Configuration Objects
+You can use this command to edit the configuration after installation: for example, the webhook port.
+You can edit configuration objects with:
+[source,terminal]
+----
+$ oc -n [namespace] edit cm hw-event-proxy-operator-manager-config
+----
+[source,terminal]
+----
+apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+kind: ControllerManagerConfig
+health:
+  healthProbeBindAddress: :8081
+metrics:
+  bindAddress: 127.0.0.1:8080
+webhook:
+  port: 9443
+leaderElection:
+  leaderElect: true
+  resourceName: 6e7a703c.redhat-cne.org
+----
+[discrete]
+
+== Project
+link:https://github.com/redhat-cne/hw-event-proxy-operator[hw-event-proxy-operator] 
+
+== CRD
+The proxy enables applications running on bare-metal clusters to respond quickly to Redfish hardware changes and failures such as breaches of temperature thresholds, fan failure, disk loss, power outages, and memory failure, reported using the HardwareEvent CR.
```

Comment:
Suggest move this description under `hardwareevents.event.redhat-cne.org`:
```suggestion
This defines defines the `Baremetal Hardware Event proxy`, which enables applications running on bare-metal clusters to respond quickly to Redfish hardware changes and failures such as breaches of temperature thresholds, fan failure, disk loss, power outages, and memory failure.
```

Reply:
Done.